tools/minimax_tts/manifest.yaml

```
@@ -0,0 +1,33 @@
+author: devin

```

```suggestion
author: langgenius
```

The author name of the plugins in this repo should be `langgenius`. If you would like to have other name, you should submit a PR to this repo https://github.com/langgenius/dify-plugins/pulls instead.

---

done